### ISSUE TITLE
Python scripts enhacement

### ISSUE THREAD

**Issue by ManuelJNunez** [rticommunity/rticonnextdds-examples#404]
## Describe your enhancement

The scripts of CI/CD retrieve return code 0 when there is an error.

```python
if not rti_minimal_package_url:
        print(
            "Environment variable RTI_MIN_PACKAGE_URL not found, skipping..."
        )
        sys.exit()
```

To fix this, we could do this refactoring:

```python
if not rti_minimal_package_url:
        sys.exit(
            "Environment variable RTI_MIN_PACKAGE_URL not found, skipping..."
        )
```
By doing this, we can simplify the new workflow.

**Comment by ManuelJNunez** [comment]
Already closed on develop. When merging to master it will be closed automatically.

**Comment by lulivi** [comment]
Already fixed in #406
